mobile/src/main/java/org/openhab/habdroid/background/ItemsControlsProviderService.kt

```diff
@@ -186,7 +186,8 @@ class ItemsControlsProviderService : ControlsProviderService() {
         }
 
         fun maybeCreateControl(item: Item): Control? {
-            if (item.label.isNullOrEmpty() || item.readOnly) return null
+            val label = item.label
+            if (label.isNullOrEmpty() || item.readOnly) return null
```

Comment:
This change doesn't look like it's needed and/or really helpful?

Reply:
It is required. `label` is now a nullable `String?` property, so no smart cast to `String` is possible anymore. Below `Control.StatefulBuilder().setTitle()` wants a non-null String.